components/ConfirmationModal.js

```diff
@@ -98,7 +98,7 @@ ConfirmationModal.propTypes = {
   /** body of the confirmation modal */
   children: PropTypes.node,
   /** Body of the confirmation modal, used in a paragraph if there's no children */
-  body: PropTypes.string,
+  body: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
```

Comment:
Good move to change this, however object is not a good type for this. If people pass `body={{ hello: 42 }}` the component will crash. What you want here is `PropTypes.node`, as per:
```
  // Anything that can be rendered: numbers, strings, elements or an array
  // (or fragment) containing these types.
```

(same applies for `continueLabel`/`cancelLabel`)

Reply:
This one wasn't addressed